addons/web_editor/static/src/js/common/ace.js

```diff
@@ -911,8 +911,11 @@ var ViewEditor = Widget.extend({
      *
      * @private
      */
-    _onSaveClick: function () {
-        this._saveResources();
+    _onSaveClick: function (ev) {
+        this.trigger_up('btn_spin_request', {btn: ev.currentTarget, spin: true});
+        this._saveResources().guardedCatch(function () {
+            this.trigger_up('btn_spin_request', {btn: ev.currentTarget});
```

Comment:
`this` is undefined here, you need to use the arrow syntax `() => {`

Reply:
Good catch, as at the end it did a page reload didn't noticed the spin didn't stopped ;-)